src/websocket.c

```diff
@@ -370,8 +370,8 @@ void ws_read_cb(uv_link_t *l, ssize_t nread, const uv_buf_t *buf) {
             ws->read_cb((uv_stream_t *) ws, UV_EOF, buf);
             break;
         case OpCode_Ping:
-            UM_LOG(TRACE, "got ping");
-            send_pong(ws, dp, len);
+            UM_LOG(TRACE, "got ping masked=%d len=%d", masked, len);
+            send_pong(ws, dp, len, masked, mask);
```

Comment:
we should not use server masked flag here.
according to https://tools.ietf.org/id/draft-ietf-hybi-thewebsocketprotocol-09.html#rfc.section.4.3
all client frames have to be masked